<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">centralwidget
{
	background:#333
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,2,1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="codeLabel">
            <property name="text">
             <string>代码</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="codeBrowser">
            <property name="styleSheet">
             <string notr="true">codeBrower{
qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255))
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="resultLable">
            <property name="text">
             <string>运行结果</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="resultBrowser"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
          <item>
           <widget class="QLabel" name="varLabel">
            <property name="text">
             <string>变量信息</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="varBrowser"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="syntaxLabel">
        <property name="text">
         <string>句法与语法树</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="syntaxDisplayBroser"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
rgb(255, 49, 8)
}</string>
          </property>
          <property name="text">
           <string>清空代码 (CLEAR)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="runButton">
          <property name="text">
           <string>执行代码 (RUN)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loadButton">
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>载入代码 (LOAD)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="inputlabel">
        <property name="text">
         <string>命令输入窗口</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="codeLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="messageLabel">
        <property name="text">
         <string>提示信息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="messageLineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
